From 20dbc5d4cfb51152278a2ad4627d1db102bf9225 Mon Sep 17 00:00:00 2001
From: Adrian Negreanu <adrian.m.negreanu@intel.com>
Date: Fri, 21 Mar 2014 07:06:15 +0000
Subject: fix out-of-tree builds

version.h is -include-ed assuming that builddir is the same
as srcdir;

In file included from <command-line>:0:0:
./../../tests/../lib/check-ndebug.h:3:1: fatal error:
../../tests/../version.h: No such file or directory
 #endif
  ^

v2: Use top_builddir as suggested by Damien - builddir gets expanded
to ./ which worked accidentally since automake includes top_builddir
by default.

Signed-off-by: Adrian Negreanu <adrian.m.negreanu@intel.com> (v1)
Signed-off-by: Daniel Vetter <daniel.vetter@ffwll.ch>
---
diff --git a/tests/Makefile.am b/tests/Makefile.am
index 99a3712..ba05eac 100644
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -24,7 +24,7 @@ AM_CFLAGS = $(DRM_CFLAGS) $(CWARNFLAGS) \
 	-I$(srcdir)/.. \
 	-I$(srcdir)/../lib \
 	-include "$(srcdir)/../lib/check-ndebug.h" \
-	-include "$(srcdir)/../version.h" \
+	-include "$(top_builddir)/version.h" \
 	-DIGT_DATADIR=\""$(abs_srcdir)"\" \
 	$(NULL)
 
--
cgit v0.9.0.2-2-gbebe
